internal/pkg/cli/app_show.go

```diff
@@ -50,26 +51,42 @@ func newShowAppOpts(vars showAppVars) (*showAppOpts, error) {
 		return nil, err
 	}
 
-	return &showAppOpts{
+	opts := &showAppOpts{
 		showAppVars: vars,
 		storeSvc:    ssmStore,
 		ws:          ws,
 		w:           log.OutputWriter,
-		initDescriber: func(o *showAppOpts, enableResources bool) error {
-			var d *describe.WebAppDescriber
-			var err error
+	}
+	opts.initDescriber = func(enableResources bool) error {
```

Comment:
As we add more patterns we might want this to be a describer provider or something that can look up the right functions in a map or something like that - so we don't end up with one giant function. We needn't do it now, but just a note.

Reply:
I agree. That's a good idea!